references/detection/run_mask_rcnn_tpu.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Prerequisite: conda activate torch-xla-nightly-vision
+set -e
+
+# TPU XLA
+export TPU_IP_ADDRESS="10.2.101.2"
+export XRT_TPU_CONFIG="tpu_worker;0;$TPU_IP_ADDRESS:8470"
+
+# CPU XLA
+#export XRT_DEVICE_MAP="CPU:0;/job:localservice/replica:0/task:0/device:XLA_CPU:0"
+#export XRT_WORKERS="localservice:0;grpc://localhost:40934"
+
+export TF_CPP_VMODULE=tensor=5 # logging
+export TRIM_GRAPH_SIZE=100000
+export TRIM_GRAPH_CHECK_FREQUENCY=1000
+
+export XLA_DENSE_SCATTER_FACTOR=100
```

Comment:
What does this do?

Reply:
Magic flag suggested by Davide when I was running into scatter issues and OOMing, will check behavior without that flag.

Reply:
Got rid of all these flags and no issue at all!